azure-iot-device/azure/iot/device/iothub/pipeline/pipeline_adapter.py

```diff
@@ -5,25 +5,38 @@
 # --------------------------------------------------------------------------
 
 import logging
-from azure.iot.device.common.transport import pipeline_stages_base
-from azure.iot.device.common.transport import pipeline_ops_base
-from azure.iot.device.common.transport.mqtt import pipeline_stages_mqtt
-from azure.iot.device.iothub.transport.abstract_transport import AbstractTransport
-from azure.iot.device.iothub.transport import pipeline_stages_iothub
-from azure.iot.device.iothub.transport import pipeline_events_iothub
-from azure.iot.device.iothub.transport import pipeline_ops_iothub
+from . import constant
+from azure.iot.device.common.pipeline import pipeline_stages_base
+from azure.iot.device.common.pipeline import pipeline_ops_base
+from azure.iot.device.common.pipeline import pipeline_stages_mqtt
+from azure.iot.device.iothub.pipeline import pipeline_stages_iothub
```

Comment:
This and all imports `from azure.iot.device.iothub.pipeline` could be simplified to `from . import <module>`. I'm not the biggest fan of the weird singular `.`, but hey, it's shorter.